### PR TITLE
Fix prefix in kinds documentation

### PR DESCRIPTION
**Pull request by michalbiesek**
Only the last commit is valid

---
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/482)

### CODE REVIEW

**Review by bratpiorka — COMMENTED** [review]
:lgtm:

> Reviewed 11 of 11 files at r1.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/482#-:-MR_FJm0DkGXtJZBOj8h:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @PatKamin)

File: man/memkindallocator.3 [hunk 1]
```diff
@@ -86,9 +86,9 @@ in the same local domain that have the lowest latency.
 Allocate from a NUMA node(s) that has the highest bandwidth among all NUMA Nodes from the local domain.
 NUMA Nodes have the same local domain for a set of CPUs associated with them, e.g. socket or sub-NUMA cluster.
 .PP
-.B libmemkind::kinds::MEMKIND_HIGHEST_BANDWDITH_LOCAL_PREFERRED
+.B libmemkind::kinds::HIGHEST_BANDWDITH_LOCAL_PREFERRED
```
Comment (PatKamin) [inline man/memkindallocator.3:89]: BANDWIDTH

File: man/memkindallocator.3 [hunk 2]
```diff
@@ -86,9 +86,9 @@ in the same local domain that have the lowest latency.
...
 Same as
-.B HIGHEST_BANDWDITH_LOCAL
+.B libmemkind::kinds::HIGHEST_BANDWDITH_LOCAL
```
Comment (PatKamin) [inline man/memkindallocator.3:91]: Same here

**Review by PatKamin — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/482)* status: all files reviewed, 3 unresolved discussions (waiting on @michalbiesek)

___
*[man/memkindallocator.3, line 85 at r1](https://reviewable.io/reviews/memkind/memkind/482#-MR_HXXw8eAaBMMXh1O3:-MR_HXXw8eAaBMMXh1O4:b-xkmac) ([raw file](https://github.com/memkind/memkind/blob/717ed4967cf317850ebe577c960499874b2c1ab1/man/memkindallocator.3#L85)):*
> ```Roff
> in the same local domain that have the lowest latency.
> .PP
> .B libmemkind::kinds::HIGHEST_BANDWDITH_LOCAL
> ```

BANDWIDTH

**Review by michalbiesek — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/482)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @bratpiorka and @PatKamin)

___
*[man/memkindallocator.3, line 85 at r1](https://reviewable.io/reviews/memkind/memkind/482#-MR_HXXw8eAaBMMXh1O3:-MR_O_r00ES2ej0XVdLa:b-896fix) ([raw file](https://github.com/memkind/memkind/blob/717ed4967cf317850ebe577c960499874b2c1ab1/man/memkindallocator.3#L85)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

BANDWIDTH
</blockquote></details>

Done.
___
*[man/memkindallocator.3, line 89 at r1](https://reviewable.io/reviews/memkind/memkind/482#-MR_Ehz9CF6_424W22Mg-r1-89:-MR_OaZaBa_IP6LuxMiA:b-896fix) ([raw file](https://github.com/memkind/memkind/blob/717ed4967cf317850ebe577c960499874b2c1ab1/man/memkindallocator.3#L89)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

BANDWIDTH
</blockquote></details>

Done.
___
*[man/memkindallocator.3, line 91 at r1](https://reviewable.io/reviews/memkind/memkind/482#-MR_Ehz9CF6_424W22Mg-r1-91:-MR_Ob888X_-4Lipzk9g:b-896fix) ([raw file](https://github.com/memkind/memkind/blob/717ed4967cf317850ebe577c960499874b2c1ab1/man/memkindallocator.3#L91)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

Same here
</blockquote></details>

Done.

**Pull request merged**